### PR TITLE
empathy: enchant2

### PR DESCRIPTION
**Pull request by Gottox**
Followup to https://github.com/void-linux/void-packages/pull/28963

### CODE REVIEW

File: srcpkgs/empathy/template [hunk 1]
```diff
@@ -24,8 +25,19 @@ license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="http://live.gnome.org/Empathy"
 distfiles="${GNOME_SITE}/$pkgname/${version%.*}/$pkgname-$version.tar.xz"
 checksum=7d86942ce97edd10ade0e6ae6a210d35e4d627fe4d223377d71fd1840bc6e3a3
+patch_args=-Np1
 disable_parallel_build=yes
 
+do_check() {
+	# Disable checks, the only work with an irc server in place.
+	:
+}
```
Comment (ericonr) [inline srcpkgs/empathy/template:34]: You can put `make_check=no` in the template directly.

**Review by ericonr — COMMENTED** [review]
What did you change from my attempt to fix it? Thanks :)

**Comment by Gottox** [comment]
`yelp-tools` is used while autoreconf. So this package was missing in hostmakedepends.

File: srcpkgs/empathy/template (hunk 1 above)
Comment (Gottox) [reply srcpkgs/empathy/template]: fixed.

**Pull request merged**